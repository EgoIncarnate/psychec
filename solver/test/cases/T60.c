void f() {
  int a = g(9.9);
  T b = 9.9;
  b = h(1, "char");
}
