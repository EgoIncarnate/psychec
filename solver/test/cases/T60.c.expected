typedef  double T ;
int g (double) ;
double h(int,char const*) ;
